src/main/kotlin/io/github/mojira/arisa/registry/LazyModuleRegistry.kt

```diff
@@ -0,0 +1,28 @@
+package io.github.mojira.arisa.registry
+
+import com.uchuhimo.konf.Config
+import io.github.mojira.arisa.infrastructure.config.Arisa
+import io.github.mojira.arisa.modules.UpdateLinkedModule
+import java.time.Instant
+import java.time.temporal.ChronoUnit
+
+/**
+ * This class is the registry for modules that get executed "lazily", i.e. with a delay of multiple hours.
+ * This is done in order to avoid spam.
+ */
+class LazyModuleRegistry(config: Config) : ModuleRegistry(config) {
```

Comment:
"Delayed" instead of "lazy" might be more fitting.

However this registry is currently specifically created for `UpdateLinkedModule`, and also has a `updated > ${lastRun.toEpochMilli()}` check which makes at least setting Linked to 0 non-delayed.
Maybe such a generic name as `LazyModuleRegistry` is therefore misleading.

Reply:
`DelayedModuleRegistry` exists already; it's used for the `DuplicateMessageModule`, which is run slightly delayed (a few minutes only).

Yeah, the name is probably not optimal. These three registries are just kind of a workaround for different requirements by the different modules.

IMO it would probably make sense to either split the part of the `UpdateLinkedModule` that only sets Linked to 0 into a separate module that would only be responsible for that, or to not have that part at all and have Linked set to 0 by default on JIRA side if that's possible to set up.

Reply:
Renamed it to `LinkedModuleRegistry`